ansible_collections/arista/avd/examples/single-dc-l3ls/README.md

```diff
@@ -0,0 +1,734 @@
+
+```yml
+- name: Run AVD
+  hosts: FABRIC
+  ansible.builtin.gather_facts: false
+  tasks:
+    - name: generate intended variables
+      import_role:
+        name: arista.avd.eos_designs
+    - name: generate device intended config and documentation
+      import_role:
+        name: arista.avd.eos_cli_config_gen
+    - name: deploy configuration to device
+      import_role:
+         name: arista.avd.eos_config_deploy_eapi
+```
+
+The name of the playbook is set along with the scope of the playbook, which in this example is the entire `FABRIC`. If you recall, this is a group name defined in the inventory, hence if you want your playbook to only apply to a subset of devices, you can change it here.
+
+This playbook has three tasks that use three roles:
+
+- `generate intended variables` uses the role `arista.avd.eos_designs`, which will generate structured configuration for each device. This structured configuration can be found in the `ansible-avd-examples/single-dc-l3ls/intended/structured_configs` folder.
+- `generate device intended config and documention` uses the role `arista.avd.eos_cli_config_gen`, which generates the actual Arista EOS CLI configurations found in the `ansible-avd-examples/single-dc-l3ls/intended/configs` folder, along with the device specific and fabric wide documentation found in the `ansible-avd-examples/single-dc-l3ls/documentation/` folder. It relies on the structured configuration generated by `arista.avd.eos_designs`.
+- Finally `deploy configuration to device` uses the role `arista.avd.eos_config_deploy_eapi` that pushes the generated configuration to the devices in scope.
+
+Please look through the folders and files described above to learn more about the output generated by Ansible AVD.
+
+### Executing the playbook
+
+The execution of the playbook should show you the following output:
+
+```shell
+user@ubuntu:~/Documents/git_projects/ansible-avd-examples/single-dc-l3ls$ ansible-playbook playbook.yml
+
+PLAY [Run AVD] *****************************************************************************************************************************************************************************
+
+TASK [arista.avd.eos_designs : Collection arista.avd version 3.5.0 loaded from /home/user/.ansible/collections/ansible_collections] ******************************************************
+ok: [dc1-leaf1a]
+
+TASK [arista.avd.eos_designs : Create required output directories if not present] **********************************************************************************************************
+ok: [dc1-leaf1a -> localhost] => (item=/home/user/Documents/git_projects/ansible-avd-examples/single-dc-l3ls/intended/structured_configs)
+ok: [dc1-leaf1a -> localhost] => (item=/home/user/Documents/git_projects/ansible-avd-examples/single-dc-l3ls/documentation/fabric)
+(...)
+```
+
+If you are not seeing a similar output to the showed one, make sure:
+
+1. You have `ansible-core` and not `ansible` installed.
```

Comment:
Installing the full "ansible" package can be ok, since it also contains ansible-core.
```suggestion
1. You meet the documented [requirements](../../docs/installation/requirements.md) 
```